### ISSUE TITLE
Failing to install plugins

### ISSUE THREAD

**Issue by derekherman** [gtap-dev/gravityforms-composer-installer#10]
Today without any changes to the Composer file the main GF plugin and add-ons are both failing to install with messages like the one below. I came across this issue while deploying an unrelated change to staging. This was working fine Tuesday when I deployed to staging other unrelated code. For content the CI/CD pipeline installs the plugins for a WP site using Composer. This is failing both locally and on our Cloud Build pipeline on GCP and I've asked others to try to run it on their machines without any success. Could someone please confirm they can install the GF plugin using composer?

```
  - Installing gravityforms/gravityforms (2.4.21.3): Downloading (100%)         
    Failed to execute unzip -qq  'gravityforms/c10a3d74075262ddef1aad1a43b02024.php' -d '{LOCAL_PATH_REMOVED}/vendor/composer/57e2916a'

[gravityforms/c10a3d74075262ddef1aad1a43b02024.php]
  End-of-central-directory signature not found.  Either this file is not
  a zipfile, or it constitutes one disk of a multi-part archive.  In the
  latter case the central directory and zipfile comment will be found on
  the last disk(s) of this archive.
unzip:  cannot find zipfile directory in one of gravityforms/c10a3d74075262ddef1aad1a43b02024.php or
        gravityforms/c10a3d74075262ddef1aad1a43b02024.php.zip, and cannot find gravityforms/c10a3d74075262ddef1aad1a43b02024.php.ZIP, period.

    The archive may contain identical file names with different capitalization (which fails on case insensitive filesystems)
    Unzip with unzip command failed, falling back to ZipArchive class

    Invalid zip file, retrying...
```

**Comment by dsturm** [comment]
Hey @derekherman,

yep, same here. I'll have a look if I can identify the cause.

---

Just for the record: Which composer version are you running? I'm facing the issue under

```sh
Composer version 2.0.3 2020-10-28 15:50:55
# and
Composer version 2.0.2 2020-10-25 23:03:59
# and
Composer version 1.10.16 2020-10-24 09:55:59
```

----

Ok, this seems to be an issue because of the changes in [`ffraenz/private-composer-installer` v5](https://github.com/ffraenz/private-composer-installer/pull/25). There have been several renames and structural changes to be compatible with Composer v2, that have not been considered in #9. I already ~set up a fix-branch which addresses this issue, but currently it fails on the first require~ published a PR (#11) to fix this.

**Comment by mihkeleidast** [comment]
I've removed the broken 2.2.0 version from packagist until the fix gets in.

**Comment by mihkeleidast** [comment]
Fixed in #11, released as 2.3.0
